code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -24,6 +24,8 @@
 	var/cooldown = 3.5 SECONDS
 	/// the time it takes before the target falls over
 	var/knockdown_delay = 2.5 SECONDS
+	/// how big the baton is while on
+	var/w_class_on = WEIGHT_CLASS_HUGE
```

Comment:
this should be making use of `can_enter_storage` not editing its weight class.

Reply:
Yeah I'll change this when I get a chance, been to busy with work to have time to code or if I do have the time I don't have to motivation,even if it's a small tweak. I've got a day off Tuesday I'll try and do it then.